cms/static/cms/sass/mixins/_buttons.scss

```diff
@@ -68,4 +68,7 @@
             }
         }
     }
+    &.cms-btn-disabled {
+        color: $gray-lighter;
```

Comment:
that's going to work only for default buttons, not for action / danger ones.
maybe it's better to fix line 63 in this file to be more transparent?